tests/admin_docs/test_views.py

```diff
@@ -337,6 +337,21 @@ def test_simplify_regex(self):
             (r'^(?P<a>(x|y))/b/(?P<c>\w+)ab', '/<a>/b/<c>ab'),
             (r'^(?P<a>(x|y)(\(|\)))/b/(?P<c>\w+)ab', '/<a>/b/<c>ab'),
             (r'^a/?$', '/a/'),
+            (r'^(?P<a>\w+)\.b', '/<a>.b'),
```

Comment:
I'm seeing only 12 of these items failing without the patch. Adding additional test coverage is okay, but please do it in a separate commit that precedes the bug fix so it's clear which cases this patch fixes.